src/test/java/io/logz/LogzioSenderTest.java

```diff
@@ -57,19 +57,19 @@ public void simpleAppending() throws Exception {
         String message1 = "Testing..";
         String message2 = "Warning test..";
 
-        Logger testLogger = createLogger(token, type, loggerName, drainTimeout, null, null, null);
+        Logger testLogger = createLogger(token, type, loggerName, drainTimeout, null, null, null, false, null);
```

Comment:
Why createLogger can't set it to false and null by default thus preventing changing it all over the place (copy paste)